Add CLI flag information for Shared Storage (#3899)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/shared-storage/index.md

{% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/CWfgCMJQ5cYPOfjttF3k.png", alt="Set Privacy Sandbox Ads APIs experiment to enabled to use these APIs", width="744", height="124" %}

> Shared Storage can be also enabled with the `--enable-features=PrivacySandboxAdsAPIsOverride,SharedStorageAPI,FencedFrames` flag from the command line. 

Reviewer: ```suggestion
You can also enable Shared Storage with the `--enable-features=PrivacySandboxAdsAPIsOverride,SharedStorageAPI,FencedFrames` flag in the command line. 
```
